client/src/App.js

```diff
@@ -19,6 +20,7 @@ function App() {
           <Route path="/" exact component={Home} />
           <Route path="/books" component={Books} />
           <Route path="/addnewbook" component={AddNewBook} />
+          <Route path="/apibook" component={AddApiBook} />
```

Comment:
I might rename this path/component as it is not clear from the name what this is supposed to do or how it differs from `/addnewbook.`